app/src/main/res/xml/remote_config_defaults.xml

```diff
@@ -233,4 +233,8 @@
         <key>contactsContentJson</key>
         <value>[{"title":"Máte podezření na koronavirus?","text":"Zůstaňte doma a kontaktujte svého praktického lékaře nebo příslušnou hygienickou stanici.","linkTitle":"Důležité kontakty","link":"https://koronavirus.mzcr.cz/dulezite-kontakty-odkazy/"},{"title":"Potřebujete poradit?","text":"Veškeré otázky a nejasnosti týkající se koronaviru najdete na webu MZČR.","linkTitle":"Často kladené otázky","link":"https://koronavirus.mzcr.cz/otazky-odpovedi/"},{"title":"O aplikaci eRouška","text":"Potřebujete poradit s aplikací nebo vás zajímá, jak funguje? Napište Anežce, podpoře eRoušky.","linkTitle":"Anežka - podpora eRoušky","link":"https://erouska.cz/#chat-open"}]</value>
     </entry>
+    <entry>
+        <key>currentMeasuresUrl</key>
+        <value>https://koronavirus.mzcr.cz/aktualni-opatreni/</value>
+    </entry>
 </defaultsMap>
```

Comment:
Přidáš ještě do remote configu? Je domluveno s iOSákama, že se takto bude jmenovat klíč k URL?

Reply:
do Remote Config jsem to už dodal a iOS týmu jsem to psal a potvrdili mi to